### ISSUE TITLE
[C][Fx58] buttons_on_navbar_squared_buttons Stop-Reload button animation

### ISSUE THREAD

**Issue by Acid-Crash** [Aris-t2/CustomCSSforFx#20]
When enabled Stop-Reload button animation jumps to the left and returns back when animation is finished.
Affected on all modes.
Most noticeable on "tablet" mode.
![reload](https://user-images.githubusercontent.com/32600318/32326490-a9e27f7e-bfdb-11e7-9369-5e4a82d2d35a.png)

**Comment by Aris-t2** [comment]
Will be fixed on next update.

Add this to your code for now.

```css
/* reload/stop button animation */
#main-window[uidensity=compact] #nav-bar #stop-reload-button .toolbarbutton-animatable-box {
  margin-left: 7px !important;
  margin-right: 7px !important;
}
#main-window:not([uidensity=compact]):not([uidensity=touch]) #nav-bar #stop-reload-button .toolbarbutton-animatable-box {
  margin-left: 11px !important;
  margin-right: 11px !important;
}
#main-window[uidensity=touch] #nav-bar #stop-reload-button .toolbarbutton-animatable-box {
  margin-left: 14px !important;
  margin-right: 14px !important;
}
```

**Comment by Acid-Crash** [comment]
fixed in 1.2.0
